test/run-drun/heartbeat.mo

```diff
@@ -22,7 +22,7 @@ actor {
      while (count < max) {
        ignore await raw_rand(); // yield to scheduler
        attempts += 1;
-       if (attempts >= 50 and count == 0)
+       if (attempts >= 200 and count == 0)
```

Comment:
Spurious change? Revert?

Reply:
yes, I have disabled the `ic-ref` test path

Reply:
200 looks like a good compromise